ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -236,19 +236,19 @@
       #end
       : $doc.display('participants')
       ## groups
-      ; &lt;label for="Meeting.Code.MeetingClass_0_groups"&gt;$!services.icon.render('group') $services.localization.render('contrib.meeting.field.groups')&lt;/label&gt;##
-      #if ($xcontext.action == 'edit')
+      #if ($xcontext.action == 'edit' || $doc.getValue('groups') != '')
+        ; &lt;label for="Meeting.Code.MeetingClass_0_groups"&gt;$!services.icon.render('group') $services.localization.render('contrib.meeting.field.groups')&lt;/label&gt;##
```

Comment:
The `label` tag is not a self closing tag as per https://www.w3.org/TR/html/sec-forms.html#the-label-element. Please fix this and properly indent it.